libpod/container_api.go

```diff
@@ -373,8 +401,11 @@ func (c *Container) Attach(noStdin bool, keys string, attached chan<- bool) erro
 	}
 
 	resize := make(chan remotecommand.TerminalSize)
-	defer close(resize)
-
+	if terminal.IsTerminal(int(os.Stdin.Fd())) {
+		resizeTty(resize)
```

Comment:
Does this mean we never close the resize?

Reply:
the ownership passes to `resizeTty` that will close the channel